document-generation/smartdocuments/src/main/kotlin/com/ritense/smartdocuments/plugin/SmartDocumentsPlugin.kt

```diff
@@ -55,13 +57,13 @@ class SmartDocumentsPlugin(
 
     @URL
     @PluginProperty(key = "url", required = true, secret = false)
-    private lateinit var url: String
+    lateinit var url: String
```

Comment:
The private can stay, right?

```suggestion
    private lateinit var url: String
```